calcom/cal.com
# fix: mark people invited to an org as email-verified

## What does this PR do?

This PR marks users invited to an org as email verified.

Fixes #11209 

## Requirement/Documentation

## Type of change




## How should this be tested?

- Invite a member to an org. 
- Try to login as the invited member. 
- You should not see email verified banner at top.

## Mandatory Tasks


## Checklist

- I haven't added tests that prove my fix is effective or that my feature works
